### ISSUE TITLE
attribute name

### ISSUE THREAD

**Issue by rviktor87** [katawohl/orientation-js-retraining#3]
https://github.com/katawohl/orientation-js-retraining/blob/5ebb866b8ba19318e4dbd6aa16b7593f2ed7db6b/golden-acorn/src/Button.js#L10

it should be `aria-label` instead of `ariaLabel`
